WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -304,10 +307,15 @@ class LoginEmailViewController: LoginViewController, SigninKeyboardResponder {
     }
 
     override func displayRemoteError(_ error: Error!) {
-        configureViewLoading(false)
+        if awaitingGoogle {
+            let socialErrorVC = LoginSocialErrorViewController(title: NSLocalizedString("Unable To Connect", comment: "Shown when a user logs in with Google but it subsequently fails to work as login to WordPress.com"), description: error.localizedDescription)
+            navigationController?.setViewControllers([socialErrorVC], animated: true)
```

Comment:
Is this a good place to set `awaitingGoogle` back to `false`?

Reply:
Yes, tho I expect this VC will get dealloced because the navigation controller is about to release it.